doc/src/quickstart.md

```diff
@@ -1,9 +1,19 @@
 # Quickstart Guide
 
-This is a short guide to get you started. For a more detailed explanation about
-what's going on, please refer to the [tutorial](tutorial/index.html).
+This is a short guide to get you started with lrpar and shows how to generate a
+standalone parser from a given grammar. lrpar is meant to be used as a
+replacement for the parser generator
+[Yacc](https://en.wikipedia.org/wiki/Yacc), and thus uses the same syntax
+whereever possible. The main difference are the grammar actions, which unlike
+Yacc, need to be written in Rust. The input for lrpar is a lexer and a grammar
```

Comment:
lrpar takes in only a grammar. lrlex takes in the lexer.

Reply:
I meant that `parse()` takes a lexer as input, but let me see how I can rewrite this.

Reply:
Ah, yeah, this is hard to phrase. Maybe "lrpar takes as input a lexer instance (from which it consumes lexemes) and a grammar (which it parses the lexemes against)" or similar?